nonmoodlefeedback.php

```diff
@@ -72,6 +73,7 @@
         [
             'userid' => $userid,
             'currenttab' => 'feedback',
+            'sesskey' => sesskey()
```

Comment:
do we really need sesskey to view the report?

Reply:
yes, it returns error in the index.php file if not, we can look into how to remove it from index.php if needed